### DISCUSSION TITLE
Volume buttons are not set to Call Volume

### DISCUSSION THREAD

**Discussion by DoKoB0512** [juha-h/baresip-studio#302] [Q&A]
So after hours of trial and error, I am here to discuss the issue regarding : 

-> In Android 13, during the sip call the volume buttons does not change the volume of call that the UI indicates instead the volume is bind to Media Volume which the volume button ignores.

-> Attaching the screenshot below for better understanding of the issue.

![image](https://user-images.githubusercontent.com/123281216/216531980-d6b9dda0-07ff-433a-8683-6c5919fab361.png)

**Comment by juha-h** [comment]
I don't have any Android 13 devices to test, but baresip is using media for audio.

**Reply by DoKoB0512** [reply]
Okay, So how can I set the volume buttons to change the audio of media volume (if baresip is using media) instead of changing the call volume? OR can I change the baresip config so that it uses the call audio volume instead of media audio?

**Comment by juha-h** [comment]
On my Android 12 device, when I during call press vol +/- buttons, they by default change Media volume.

**Comment by DoKoB0512** [comment]
Yes that's correct, I've tried with Android 11 and 12 and it works fine for them. I've tried the speaker ON - OFF and that works fine too. It's just the Android 13 that's bugging me. I think I need to check for some work-around.
